docs/UserGuide.md

```diff
@@ -145,34 +145,51 @@ Format: `find PREFIX KEYWORD [MORE_KEYWORDS]`
   e.g. `chicken steak` will return `chicken steak`, `steak beef`
 
 Examples:
-* `find n/Chicken` returns `chicken` and `CHICKEN`
+* `find-i n/Chicken` returns `chicken` and `CHICKEN`
 
 
 
-### Deleting an item : `delete`
+### Deleting an item : `delete-i`
 
 Deletes the specified item from the inventory book.
 
-Format: `delete INDEX`
+Format: `delete-i INDEX`
 
 * Deletes the item at the specified `INDEX`.
 * The index refers to the index number shown in the displayed item list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd item in the inventory book.
-* `find Duck` followed by `delete 1` deletes the 1st item in the results of the `find` command.
+* `list-i` followed by `delete-i 2` deletes the 2nd item in the inventory book.
+* `find-i Duck` followed by `delete-i 1` deletes the 1st item in the results of the `find-i` command.
 
 
 
-### Clearing all entries : `clear`
+### Clearing all entries : `clear-i`
 
 Clears all entries from the Inventory book.
 
-Format: `clear`
+Format: `clear-i`
 
 
 
+### Locating deliveries by keywords: `find-d`
+
+Finds deliveries whose attributes contain any of the given keywords.
+
+Format: `find-d PREFIX KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `chicken` will match `CHICKEN`
+* The order of the keywords does not matter. e.g. `Chicken steak` will match `steak Chicken`
+* Name, Supplier and Tag can be searched
+* Only full words will be matched e.g. `chicke` will not match `chicken`
```

Comment:
```suggestion
* Only full words will be matched e.g. `chicken` will not match `chicken`
```

Reply:
"will NOT match" so original should be fine

Reply:
Oops sorry didnt read properly, thought it was a typo!